src/services/MailService.js

```diff
@@ -2,7 +2,7 @@ import {createTransport} from 'nodemailer'
 import smtpConfig from '../smtpConfig'
 import {compileFile} from 'pug'
 
-const develop = true
+const develop = process.env.NODE_ENV === 'develop'
```

Comment:
Check here for 'development'. See here for the other possible values for `NODE_ENV`: https://canho.me/2016/07/20/the-hidden-powers-of-node-env/

Reply:
Done.